crypto/signer.go

```diff
@@ -20,23 +20,23 @@ const (
 	mapKeyTreeSize       string = "TreeSize"
 )
 
-// TrillianSigner is responsible for signing log-related data and producing the appropriate
+// Signer is responsible for signing log-related data and producing the appropriate
```

Comment:
This rename possibly increases the confusion with objects in the golang crypto package. e.g. Signer has a field called signer that's a crypto.Signer.